plugins/modules/proxmox_backup.py

```diff
@@ -0,0 +1,751 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright John Berninger (@jberning) <john.berninger at gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: proxmox_backup
+short_description: Create, delete, or update Proxmox VE backup jobs
+version_added: 9.1.0
+description:
+  - Allows you to perform some supported operations on a backup job in a Proxmox VE cluster.
+author: Dylan Leverrier (@zerchevack)
+attributes:
+  check_mode:
+    support: full
+  diff_mode:
+    support: full
+options:
+  state:
+    description:
+      - Set to V(present) to create or update job.
+      - Set to V(absent) to delete job.
+    choices: ['present', 'absent']
+    type: str
+    required: true
+  all:
+    description:
+      - Backup all known guest systems on this host.
+      - Can not be use with O(vmid) and O(pool) in same job
+    type: bool
+  bwlimit:
+    description:
+      - Limit I/O bandwidth (in KiB/s).
+    type: int
+  comment:
+    description:
+      - Description for the Job.
+    type: str
+  compress:
+    description:
+      - >
+        If you choice a renote storage (like Proxmox Backup Server storage) the V(zstd) will be set automatically and this the only available value.
+        If you choice a local storage you can choice between V(gzip), V(lzo) and V(zstd).
+    choices: ['gzip', 'lzo', 'zstd']
+    type: str
+  dow:
+    description:
+      - Day of week selection.
+    type: str
+  dumpdir:
+    description:
+      - Store resulting files to specified directory.
+    type: str
+  enabled:
+    description:
+      - Enable or disable the job.
+    type: bool
+  exclude:
+    description:
+      - Exclude specified guest systems (assumes --all)
```

Comment:
```suggestion
      - Exclude specified guest systems (assumes V(--all)).
```
Or do you mean the `all` option of this module, set to `true`?
```suggestion
      - Exclude specified guest systems (assumes O(all=true)).
```

Reply:
Fix in 53df0504d36dc7ee70c350ae3467cca413e0a8b0